packages/flutter_migrate/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.0.1+4
 
+* Added `missing_code_block_language_in_doc_comment` lint.
```

Comment:
Nit: Adds

(Per the CHANGELOG style guide in the contributing docs.)

Reply:
Same for the other CHANGELOGs as well.

Reply:
Done, thanks for the reminder!